tests/generate.ts

```diff
@@ -128,10 +128,9 @@ testProcessProject(
 )
 
 testProcessProject(
-  'generates type guards for empty object',
```

Comment:
Not sure if we needed to delete this test case... But I guess it's covered by the `if exportAll is true` test.